base_station/gui/src/components/ArmControls.vue

```diff
@@ -6,6 +6,9 @@
     <div class="keyboard">
       <GimbalControls/>
     </div>
+    <div style="font-color: abs(wristTurnCount) >= 2 ? red : black;">
```

Comment:
@catkims this doesn't seem to work properly, not sure the syntax to fix

Reply:
This shouldn't block a merge though

Reply:
I'm looking into it now